bin/uglifyjs

```diff
@@ -401,7 +401,7 @@ async.eachLimit(files, 1, function (file, cb) {
         writeNameCache("props", cache);
     })();
 
-    var SCOPE_IS_NEEDED = COMPRESS || MANGLE || BEAUTIFY || ARGS.lint;
+    var SCOPE_IS_NEEDED = true;
```

Comment:
Yeah, figure_out_scope() is required everywhere now.  I could drop SCOPE_IS_NEEDED if you want.
Please feel free to suggest or apply patches as I'm not familiar with the entire code base.